api/src/main/java/org/openmrs/Person.java

```diff
@@ -106,6 +108,8 @@
 	
 	@Transient
 	private Map<String, PersonAttribute> allAttributeMap = null;
+
+	private Set<PersonAttribute> attributes;
```

Comment:
We don't want this. Any time we need to access attributes we should make a call to `getAttributes()`

```suggestion
```